index.html

```diff
@@ -1,19 +1,83 @@
 <!doctype html>
+  <body>
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+        <link rel="stylesheet" href="css/index.css">
+        <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+        <!--[if lt IE 9]>
+          <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+        <![endif]-->
+    </head>
 
-  <title>My Resume</title>
+      <div class="container">
+        <section class="top-content">
+          <h1>Chazaray J. Robinson</h1>
+          <nav>
+            <div class="images">
+                <img class="Profile-image" src="images/desk.jpg" alt="desk">
```

Comment:
Make sure the src of the `image` is exact. Your folder name is capitalized to `Image` instead of what you are trying to reference in this line: `images/desk.jpg` should be `Images/desk.jpg`